hack/cluster-test-gh-action.yaml

```diff
@@ -0,0 +1,31 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-config-override
+  namespace: rook-ceph
+data:
+  config: |
+    [global]
+    osd_pool_default_size = 1
+    mon_warn_on_pool_no_redundancy = false
+---
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: my-cluster
+  namespace: rook-ceph
+spec:
+  dataDirHostPath: /var/lib/rook
+  cephVersion:
+    image: ceph/ceph:v15
+    allowUnsupported: true
+  mon:
+    count: 1
+  dashboard:
+    enabled: false
+  crashCollector:
+    disable: true
+  storage:
+    useAllNodes: true
+    useAllDevices: true
+    deviceFilter: "^DEVICE."
```

Comment:
Why do we need the device filter? What if we just set useAllDevices without the filter?

Reply:
If we still need the filter, what if we just added a new line to the cluster-test.yaml instead of creating this whole new test yaml?
```yaml
# deviceFilter: "^DEVICE."
```

Reply:
> If we still need the filter, what if we just added a new line to the cluster-test.yaml instead of creating this whole new test yaml?

Then the sed becomes a bit more difficult 👍 
> 
> ```yaml
> # deviceFilter: "^DEVICE."
> ```

Somehow the `useAllDevices` was not working IIRC. Let me try again.

Reply:
It was failing trying to use a partition... So still using the deviceFilter